teerex/src/tests/test_cases.rs

```diff
@@ -83,6 +83,32 @@ fn register_quoting_enclave_works() {
 	})
 }
 
+#[test]
+fn register_tcb_info_works() {
+	new_test_ext().execute_with(|| {
+		Timestamp::set_timestamp(1671606747000);
```

Comment:
Could you please add some comments for this magic number?
Like so:
`// this is 2022-11-17T12:45:32Z`